--- libtommath-1.3.0/makefile.shared.orig	2024-05-30 22:06:46.315518704 +0200
+++ libtommath-1.3.0/makefile.shared	2024-05-30 22:11:31.050642828 +0200
@@ -70,7 +70,7 @@ install: $(LIBNAME)
 	install -d $(DESTDIR)$(INCPATH)
 	$(LIBTOOL) --mode=install install -m 644 $(LIBNAME) $(DESTDIR)$(LIBPATH)/$(LIBNAME)
 	install -m 644 $(HEADERS_PUB) $(DESTDIR)$(INCPATH)
-	sed -e 's,^prefix=.*,prefix=$(PREFIX),' -e 's,^Version:.*,Version: $(VERSION_PC),' -e 's,@CMAKE_INSTALL_LIBDIR@,lib,' \
+	sed -e 's,^prefix=.*,prefix=$(PREFIX),' -e 's,^Version:.*,Version: $(VERSION_PC),' -e 's,\$${prefix}/@CMAKE_INSTALL_LIBDIR@,$(LIBPATH),' \
 		-e 's,@CMAKE_INSTALL_INCLUDEDIR@,include,' libtommath.pc.in > libtommath.pc
 	install -d $(DESTDIR)$(LIBPATH)/pkgconfig
 	install -m 644 libtommath.pc $(DESTDIR)$(LIBPATH)/pkgconfig/
