diff -urNp -x '*.orig' libtommath-1.2.0.org/makefile.shared libtommath-1.2.0/makefile.shared
--- libtommath-1.2.0.org/makefile.shared	2019-10-22 09:44:44.000000000 +0000
+++ libtommath-1.2.0/makefile.shared	2022-04-04 17:54:55.505014848 +0000
@@ -70,7 +70,7 @@ install: $(LIBNAME)
 	install -d $(DESTDIR)$(INCPATH)
 	$(LIBTOOL) --mode=install install -m 644 $(LIBNAME) $(DESTDIR)$(LIBPATH)/$(LIBNAME)
 	install -m 644 $(HEADERS_PUB) $(DESTDIR)$(INCPATH)
-	sed -e 's,^prefix=.*,prefix=$(PREFIX),' -e 's,^Version:.*,Version: $(VERSION_PC),' libtommath.pc.in > libtommath.pc
+	sed -e 's,^prefix=.*,prefix=$(PREFIX),' -e 's,^libdir=.*,libdir=$(LIBPATH),' -e 's,^Version:.*,Version: $(VERSION_PC),' libtommath.pc.in > libtommath.pc
 	install -d $(DESTDIR)$(LIBPATH)/pkgconfig
 	install -m 644 libtommath.pc $(DESTDIR)$(LIBPATH)/pkgconfig/
 
